tests/unit/commands/delete/test_delete_context.py

```diff
@@ -73,26 +67,6 @@ def test_delete_context_enter(self, get_boto_client_provider_mock):
             )
         ),
     )
-    @patch("samcli.commands.delete.delete_context.click.get_current_context")
```

Comment:
Why did we remove this test? This should resolve as it was written since those are the provided values right?

Reply:
Since we're using the decorator to read the config and its values all the parsing of the config file happens there instead, and the DeleteContext just has the values passed in now instead of doing any parsing.